app/src/containers/b2b/EditAccount.tsx

```diff
@@ -111,11 +114,24 @@ export default class EditAccount extends React.Component<EditAccountProps, EditA
         },
         body: JSON.stringify({
           name,
-          'external-id': externalId,
           'legal-name': legalName,
           'registration-id': registrationNumber,
         }),
       })
+        .then(() => {
+          if (editMetadataUri) {
+            adminFetch(`${editMetadataUri}`, {
```

Comment:
General question: Is there ever a point where metadata cannot be editable?

Reply:
nope, currently there is only going to be external-id in the metadata, but seller should always be able to update what's present here. if we intend to add data that shouldn't be update it would go onto another resource.